spec/features/admin/order_cycles_spec.rb

```diff
@@ -58,6 +58,38 @@
     end
   end
 
+  describe 'listing order cycles with other locales' do
+    let!(:oc_de) { create(:simple_order_cycle, name: 'oc', orders_open_at: '2012-01-01 00:00:00') }
+
+    before { I18n.locale = :de }
+    after { I18n.locale = :en }
+
+    context 'using datepickers' do
+      it "correctly open the datepicker and changes the date field" do
+        expect(I18n.locale).to eq :de
```

Comment:
I don't see why we need assertion for this. We just set the locale to `de`. It feels like we don't trust ruby.